### PR TITLE
update benching

### PR DESCRIPTION
**Pull request by chiefbiiko**
`benching` now lives in [deno_std](https://github.com/denoland/deno_std)

### CODE REVIEW

File: src/database.json [hunk 1]
```diff
@@ -3,10 +3,6 @@
     "url": "https://raw.githubusercontent.com/zhmushan/abc/${b}/",
     "repo": "https://github.com/zhmushan/abc"
   },
-  "benching": {
-    "url": "https://raw.githubusercontent.com/chiefbiiko/benching/${b}/",
```
Comment (j-f1) [inline src/database.json]: Shouldn’t this just be updated to reference the library in `deno_std`?

Comment (chiefbiiko) [reply src/database.json]: I don't know. Wasn't even thinking of that. Should it?

Comment (ry) [reply src/database.json]: Yes please

Comment (chiefbiiko) [reply src/database.json]: :pray:
updated

**Pull request merged**